gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/service/JavaServiceDistributionExtension.java

```diff
@@ -85,7 +97,13 @@ public JavaServiceDistributionExtension(Project project) {
         excludeFromVar = objectFactory.listProperty(String.class);
         excludeFromVar.addAll("log", "run");
 
-        env = objectFactory.mapProperty(String.class, String.class).empty();
+        env = objectFactory.mapProperty(String.class, String.class);
+        env.putAll(project.provider(() -> {
+            return jdks.get().entrySet().stream()
```

Comment:
can't you map `jdks`?

Reply:
Already have - I thought you'd notice that heh